core/src/com/arco/towerdefense/game/screens/MenuScreen.java

```diff
@@ -41,53 +37,59 @@ public void render(float delta) {
 
     public void update() {
         playButtonUpdate();
+        helpButtonUpdate();
         quitButtonUpdate();
     }
 
     public void playButtonUpdate() {
-        float posX = Utils.getScreenCenterX() - playButton.getWidth()/2;
-        float posY = Utils.getScreenCenterY() - playButton.getHeight()/2;
-
-        if (Utils.isCursorInside(posX, posY, playButton.getWidth(), playButton.getHeight())) {
-            game.batch.draw(playButtonHigh, posX, posY);
+        if (Utils.isCursorInside(30, 50, 200, 145)) {
+            game.batch.draw(playButton, 30, 30,220,220);
```

Comment:
Evita esses numeros mágicos igual eu disse antes. Pega a imagem e corta ela somente pro conteúdo que você quer. Ai  vc faria algo tipo o que tava antes:
```suggestion
        if (Utils.isCursorInside(posX, posY, playButton.getWidth(), playButton.getHeight())) {
            game.batch.draw(playButtonHigh, posX, posY);
```
e isso só funciona se a imagem do playButton só tiver o conteudo do botão mesmo. Tipo a altura do botão == altura da imagem

Reply:
O objetivo é tentar evitar ao maximo esses numeros q a gente olha e fica: wtf